src/test/java/core/basesyntax/service/amount/BalanceHandlerTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.amount;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static final FruitTransaction bananaBalance;
+    private final OperationHandler balance = new BalanceHandler();
+
+    static {
+        bananaBalance = new FruitTransaction.FruitTransactionBuilder()
+                .setOperation(Operation.BALANCE)
+                .setFruitType("banana")
+                .setAmount(10)
+                .build();
+    }
+
+    @Test
+    public void balanceHandlerTest_ok() {
```

Comment:
```suggestion
    public void process_ok() {
```
naming of test methods `<(name_of_the_method_ that_is_being_tested)_(case_that_you_test(in case of such case you don't need to add this part, because you have only one test metho here))_(ok/notOk)>`, fix it in every test class